.github/workflows/ci.yml

```diff
@@ -28,9 +28,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        scala: [2.12.17]
+        scala: [2.12]
         java: [temurin@8, temurin@17]
         project: [rootJVM]
+        exclude:
```

Comment:
I don't really understand why this exclusion was added—why is the default value based on `githubWorkflowScalaVersions.value.filterNot(_ == defaultScala)`?

https://github.com/typelevel/sbt-typelevel/blob/6077b3ed02762b853f0eed482af6c3f86c39c1a9/core/src/main/scala/org/typelevel/sbt/TypelevelPlugin.scala#L79-L85

Reply:
It's because you swapped `2.12.x` for `2.12`. Let's not make that change in 0.4.x, along with the sbt upgrade. It breaks too much stuff unfortunately, like this.

Reply:
Bummer… I think that makes it impossible to dogfood one of the actual things this is intended to change. I don't want to go deeper into a rabbit hole though so I'll revert that part of it.